package.json

```diff
@@ -24,15 +24,15 @@
   },
   "peerDependencies": {
     "@alaskaairux/design-tokens": "^3.0.0",
-    "@alaskaairux/webcorestylesheets": "^3.0.0",
     "@alaskaairux/orion-icons": "^2.1.5",
+    "@alaskaairux/webcorestylesheets": "^3.0.0",
```

Comment:
```suggestion
    ""@alaskaairux/webcorestylesheets": "^3.7.3",
```

Reply:
That's the peer-dep -- I don't think that should be the same version requirements?

This bug is internal to the package and will be fixed regardless of whether the user has fully updated WCSS. They don't need to be on the latest WCSS to use the package - any v3 should be fine. Or am I missing something?

Reply:
Wait... why is the peer dependency even ther? It's not needed to run the element? All the Sass is processed into the element itself, so there is no dependency other than DEV. 

So, please remove that.

Reply:
Removed.